packages/magika.vm/magika.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>magika.vm</id>
+    <version>0.5.0</version>
+    <authors>Google</authors>
+    <description>Magika is an AI powered file type detection tool that uses deep learning to provide accurate detection.</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="python3.vm" />
```

Comment:
we need to require `libraries.python3.vm` to ensure there are not incompatibilities and both use the same version.